actions/spa-setup-task/src/spa.ts

```diff
@@ -183,18 +194,24 @@ export function spaSetupTask(
 
   const ingresses: Ingress[] = []
 
-  for (const ingress of urls) {
-    const {hostname: ingressHost, pathname: ingressPath} = new URL(ingress)
-    const {naisCluster, ingressClass} = parseIngress(ingressHost)
-
+  if (hasCustomIngressClass(ingressClass)) {
+    const {hostname: ingressHost, pathname: ingressPath} = new URL(urls[0])
     ingresses.push({ingressHost, ingressPath, ingressClass})
+    naisClusterFinal = naisCluster
+  } else {
+    for (const ingress of urls) {
+      const {hostname: ingressHost, pathname: ingressPath} = new URL(ingress)
+      const {naisCluster, ingressClass} = parseIngress(ingressHost)
+
+      ingresses.push({ingressHost, ingressPath, ingressClass})
 
-    naisClusterFinal = naisClusterFinal || naisCluster
+      naisClusterFinal = naisClusterFinal || naisCluster
 
-    if (naisClusterFinal !== naisCluster) {
-      throw Error(
-        `Ingresses must be on same cluster. Found ${naisClusterFinal} and ${naisCluster}`
-      )
+      if (naisClusterFinal !== naisCluster) {
+        throw Error(
+          `Ingresses must be on same cluster. Found ${naisClusterFinal} and ${naisCluster}`
```

Comment:
Maybe it's cool and good to have a SPADEPLOY-002: Ingresses must be on same cluster . . 🙃

Reply:
Jeg liker den idéen der, men kanskje litt prematurt før vi har et sted å lenke til? Eller, må jo starte et sted. Så kan slenge inn `SPADEPLOY-XXX` i hver feilmelding, og bare starte på toppen.

Reply:
there's no need to link! 
It's sufficient to just have a number and then people can ask "Im getting a SDAPELOY-52, no good url" but i I am sure the url is good!